packages/atomic/src/components/search/generated-answer/atomic-generated-answer-feedback/atomic-generated-answer-feedback-modal.pcss

```diff
@@ -0,0 +1,26 @@
+@import '../../../../global/global.pcss';
+
+[part='reason'] {
+  @apply text-base leading-4 text-neutral-dark;
+
+  @screen desktop-only {
+    @apply mt-6;
+  }
+
+  @screen mobile-only {
+    @apply mt-4;
+  }
+}
+
+.feedbackHeader {
+  @apply p-8;
+}
+
+[part='form'] {
+  @apply text-base leading-4 text-neutral-dark p-2 text-xl;
```

Comment:
What's the reason to use `@apply` here, as opposed to just setting them as class directly in JSX ?